understanding-ibc-denoms/index.md

```diff
@@ -0,0 +1,191 @@
+---
+parent:
+  title: Understand IBC Denoms with Gaia
+order: 0
+description: Send tokens with IBC and trace a denom, understand how denoms work.
+---
+
+# Understand IBC Denoms with Gaia
+
+One of the most powerful technologies when using the Cosmos SDK is the Interblockchain Communication Protocol (IBC). While with Cosmos every blockchain is intended to be soverign and application specific, with IBC every blockchain can connect to another blockchain using the IBC protocol. This will eventually create a system of soverign _and_ connected blockchains.
+
+## Introduction
+
+The current most used feature with IBC is sending token from one blockchain to another. When sending a token from one blockchain to another, you will end up with a token `voucher` on the other (target) blockchain.
+
+Imagine two blockchains, blockchain A and blockchain B. In the beginning, you have your token on blockchain A.
+
+![IBC token transfer](./ibc_token.png "IBC token transfer")
+*Sending token from blockchain A to blockchain B*
+
+When sending the token with IBC to another blockchain, your token will be locked up on blockchain A and create a `voucher` replacement token on blockchain B. The only way to unlock the locked tokens on blockchain A is to "send back" the `voucher` token on blockchain B. The `voucher` token on blockchain B in result will be burned.
+
+In this tutorial you will learn the format of the `voucher` token on blockchain B. What information they include, how they look like and how to make sense of them. The information of the token is described as an `IBC denom`. You can parse this IBC denom and receive information about the `voucher` and from which blockchain it came from.
+
+**You will learn how to:**
+
+- Trace the IBC denom
+- Understand how denoms work
+- Find out which chain the token came from
+
+## Requirements
+
+Install the gaia binary:
+
+```bash
+git clone https://github.com/cosmos/gaia.git
+cd gaia
+git checkout v5.0.0
+make install
+
+gaiad version
+```
+
+The output of `gaiad version` should print
+
+```bash
+v5.0.0
+```
+
+## What is this IBC denom
+
+The `voucher` token that were introduced are called IBC Denominations (IBC denom). They are the result of a token transfer using IBC from one blockchain to another. The format of the token is
+
+`ibc/DENOMHASH`.
+
+Imagine that you've received a new `ibc/` token on blockchain B where you initially held `samoleans` and `stake` token.
```

Comment:
```suggestion
Imagine that you've received a new `ibc/` token on blockchain B where you initially held `samoleans` and `stake` token.

```

Reply:
let's always use single lines, so I've added an extra space after the paragraph. Let's follow this guideline to add a blank line after each paragraph, code snippet, whatever. Following the advice in https://www.markdownguide.org/basic-syntax/